tests/cli.js

```diff
@@ -115,8 +115,10 @@ exports.group = {
         ]);
       } catch (err) {
         var msg = out.args[1][0];
+        console.log(msg);
```

Comment:
No.
I will delete it.

On 19:27, Sun, Feb 15, 2015 Rick Waldron <notifications@github.com> wrote:

> In tests/cli.js
> <https://github.com/jshint/jshint/pull/2176#discussion_r24724188>:
>
> > @@ -115,8 +115,10 @@ exports.group = {
> >          ]);
> >        } catch (err) {
> >          var msg = out.args[1][0];
> > +        console.log(msg);
>
> Is this intentional?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jshint/jshint/pull/2176/files#r24724188>.
>